modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -409,6 +417,15 @@ export default class NestedTabs extends Base {
 			this.updateListeners( view );
 			elementor.$preview[ 0 ].contentWindow.dispatchEvent( new CustomEvent( 'elementor/elements/link-data-bindings' ) );
 		}
+
+		this.resetActiveTab();
+	}
+
+	resetActiveTab() {
```

Comment:
When using the repeater functionality we sometimes loose track of the activate tab. 
Therefore I have added a function to reset the activate tab after running a repeater function.